frontend/src/App.js

```diff
@@ -1,9 +1,43 @@
 import React from 'react';
+import { BrowserRouter, Routes, Route, Navigate } from 'react-router-dom';
+import { Form } from 'components/Form';
+import { Welcome } from 'components/Welcome';
+import { Start } from 'components/Start';
+import { NotFound } from 'components/NotFound';
+import styled from "styled-components";
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <BrowserRouter>
+      <StyledMain>
+        <Routes>
+          <Route path={"/"} element={<Start />} />
+          <Route path={"/register"} element={
+            <Form
+              formType="register"
+              formTitle="Enter details"
+              buttonText="Register me!" />}
+          />
+          <Route path={"/login"} element={
+            <Form
+              formType="login"
+              formTitle="User login "
+              buttonText="Sign me in!" />}
+          />
+          <Route path={"/welcome"} element={<Welcome />} />
+          <Route path={"/404"} element={<NotFound />} />
+          <Route path={"*"} element={<Navigate to="/404" />} />
+        </Routes>
+      </StyledMain>
+    </BrowserRouter>
   );
 }
+
+const StyledMain = styled.main`
+  height: 100%;
+  display: flex;
+  height: 100%;
+  align-items: center;
+  background-color: #fdb49b;
+  background-image: linear-gradient(0deg, #fdb49b 0%, #e197d2 100%);
+`
```

Comment:
Our app.js looks kind of the same with the Routes and some styling. What we thought about was that if you keep on building an bigger app, it would probably be handy to keep the app.js with less code and instead have another component and then import it in app.